server/src/main/java/io/crate/expression/scalar/arithmetic/ArithmeticFunctions.java

```diff
@@ -131,7 +135,7 @@ public static void register(ScalarFunctionModule module) {
                     DataTypes.INTEGER.getTypeSignature(),
                     DataTypes.INTEGER.getTypeSignature(),
                     DataTypes.INTEGER.getTypeSignature()
-                ).withFeatures(op.features),
+                ).withFeatures(Sets.concat(op.features, NULLABLE)),
```

Comment:
Here and everywhere else, I agree with @mkleen's suggestion to use one more line below:
`.withFeature(NULLABLE)`

Reply:
Thanks! also updated.